lib/index.js

```diff
@@ -158,21 +158,33 @@ class Linter {
   static errorsToMessages(filePath, errors, options) {
     errors = errors || [];
     options = options || {
-      verbose: false
+      verbose: false,
+      quiet:   false
     };
 
     if (errors.length === 0) {
       return '';
     }
 
-    let errorsMessages = errors.map((error) => this._formatError(error, options)).join('\n');
+    let errorsMessages = errors
+      .map((error) => this._formatError(error, options))
+      .filter(error => error !== '')
+      .join('\n');
+
+    if (errorsMessages === '') {
+      return '';
+    }
 
     return `${chalk.underline(filePath)}\n${errorsMessages}\n`;
   }
 
   static _formatError(error, options) {
     let message = '';
 
+    if (options.quiet && error.severity === WARNING_SEVERITY) {
+      return message;
```

Comment:
TBH, I'd prefer to return something else here (especially if we are going to filter them out in `Linter.errorsToMessages`).  Perhaps `return null`?

Reply:
Yes, I wasn't really sure about this. I couldn't think of any other way to remove the warnings before it gets to the `.join('\n')` and create empty new lines everywhere. 

Ah, what about using filter to do the quiet and severity check instead?

```js
.filter((error) => {
    if (options.quiet && error.severity < ERROR_SEVERITY) {
        return false;
    }
    
    return true;
})
```

Then there are no changes to `_formatError` and it will always format an error (as the function suggests)

Reply:
Ya, sounds good to me...